public/app/plugins/datasource/grafana-pyroscope-datasource/datasource.ts

```diff
@@ -101,6 +102,25 @@ export class PhlareDataSource extends DataSourceWithBackend<Query, PhlareDataSou
       labelMatchers: extractLabelMatchers(tokens),
     };
   }
+
+  getDefaultQuery(app: CoreApp): Partial<Query> {
```

Comment:
This makes initialization in explore a bit better so for the first render of query editor we get this instead of totally empty query object.